students/kwangsoo/users/views.py

```diff
@@ -29,10 +30,13 @@ def post(self, request):
             if User.objects.filter(email = email).exists():
                 return JsonResponse({'Message' : 'REGISTERED_EMAIL'}, status = 400)
 
+            hashed_pw    = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt())
+            hashed_pw    = hashed_pw.decode('utf-8')
```

Comment:
- 디코딩을 미리하면 되지않을까요?ㅎㅎ 굳이 두줄로 나눠서 작성할 필요가 없어보입니다.
- 추가로 지금까지 전부다 `password`로 작성해주시다가 `pw`와 같은 줄임말을 사용하는 것은 좋지 않아보입니다. 간단한 변수라고 생각해서 줄이다보면 사람이란게 어디까지 줄여버릴지 모르기 때문에....
```suggestion
            hashed_password    = bcrypt.hashpw(password.encode('utf-8'), bcrypt.gensalt()).decode('utf-8')
```